src/ui/public/index_patterns/_calculate_indices.js

```diff
@@ -17,10 +17,10 @@ define(function (require) {
     function compileOptions(pattern, timeFieldName, start, stop) {
       const constraints = {};
       if (start) {
-        constraints.min_value = { gte: moment(start).valueOf() };
+        constraints.max_value = { gte: moment(start).valueOf() };
```

Comment:
I think we should just be passing the start value as is and not try to support date-formatted strings. Is that an option?